dapps/evancore.vue.libs/src/components/permissions-editor/permissions-editor.ts

```diff
@@ -109,6 +108,16 @@ class PermissionsEditor extends mixins(EvanComponent) {
     default: null
   }) sortFilters: SortFiltersInterface | string[];
 
+  /**
+   * Current users runtime.
+   */
+  runtime: bcc.Runtime;
```

Comment:
we have this.getRuntime in the EvanComponent ... not sure what's actually the best way to retrieve it. 

@Tschuck  ?

Reply:
a nvm this is just the instance ...

Reply:
The `getRuntime()` function is used to access the vuex store easier. So you can also just use the following: `this.$store.state.runtime`.